test/RPCAction.test.js

```diff
@@ -0,0 +1,39 @@
+const QueueManager = require('../src/QueueManager')
+const ConsoleInspector = require('./consoleInspector')
+let config = require('./config/LoadConfig')
+
+describe('RPCClient && RPCServer actions', function () {
+  let rpcName = 'test-rpc-action'
+  const logger = new ConsoleInspector(console)
+  let timeoutMs = 1000
+
+  const queueManager = new QueueManager(config)
+  queueManager.setLogger(logger)
+
+  let rpcClient = queueManager.getRPCClient(rpcName, {queueMaxSize: 100, timeoutMs})
+  let rpcServer = queueManager.getRPCServer(rpcName, {prefetchCount: 1, timeoutMs})
+
+  before(() => {
+    return queueManager.connect()
+  })
+
+  after(() => {
+    logger.empty()
+  })
+
+  it('RPCServer.registerAction() registers all the actions, RPCClient.callAction() sends an object and RPCServer.consume() process it', (done) => {
```

Comment:
In this test you sent a string not an object.

Reply:
fix